src/docs/development/tools/devtools/memory.md

```diff
@@ -149,86 +185,139 @@ in the legend.
 
 For more information, see [Dart VM internals][vm].
 
-## Event timeline
+## Memory Android Chart (ADB)
+
+DevTools, for an Android app, collects Android's ADB tool parsing the meminfo section of ADB's App Summary section.  This section is the most interesting from a high-level
+```
+> adb shell dumpsys meminfo io.flutter.demo.gallery -d
+
+ App Summary
+                       Pss(KB)
+                       -------
+           Java Heap:     5192
+         Native Heap:    11992
+                Code:     2132
+               Stack:       60
+            Graphics:    53700
+       Private Other:    42800
+              System:    84493
+ 
+               TOTAL:   200369       TOTAL SWAP PSS:    82168
+```
+
+This chart is another timeseries graph but of Android memory (polled ADB values).  to help visualize the state of the
+heap at various points in time.
+
+The chart's x-axis is a timeline of events (timeseries).
+In other words, it shows the polled state of the memory
+every 500 ms. This helps give a live appearance on the
+state of the memory as the application is running.
+The quantities plotted on the y-axis are (from top to bottom):
```

Comment:
looks like we are missing some information after this?

Reply:
Done.